cg/services/order_validation_service/rules/case_sample/pedigree/models.py

```diff
@@ -1,12 +1,21 @@
+from cg.services.order_validation_service.models.existing_sample import ExistingSample
+from cg.services.order_validation_service.workflows.mip_dna.models.case import MipDnaCase
+from cg.services.order_validation_service.workflows.mip_dna.models.sample import MipDnaSample
 from cg.services.order_validation_service.workflows.tomte.models.case import TomteCase
-from cg.services.order_validation_service.workflows.tomte.models.sample import (
-    TomteSample,
-)
+from cg.services.order_validation_service.workflows.tomte.models.sample import TomteSample
+from cg.store.store import Store
 
 
 class Node:
-    def __init__(self, sample: TomteSample, case_index: int, sample_index: int):
-        self.sample: TomteSample = sample
+    def __init__(
+        self,
+        sample: TomteSample | MipDnaSample | ExistingSample,
```

Comment:
Should this be an alias?

Reply:
It could be an alias defined at the top of this file. I thought there was already an alias called SampleWithRelatives somewhere

Reply:
Ohh there is! But so far it only considers new samples. I can see what happens if I add the ExistingSample there and type hint it as such